backend/src/schema/resolvers/messages.ts

```diff
@@ -13,11 +13,32 @@ export default {
           id: context.user.id,
         },
       }
+      // this does not work
+      // params.orderBy = ['createdAt_desc']
       const resolved = await neo4jgraphql(object, params, context, resolveInfo)
+
       if (resolved) {
-        resolved.forEach((message) => {
-          message._id = message.id
+        const session = context.driver.session()
+        const countMessageTxPromise = session.readTransaction(async (transaction) => {
+          const countMessageCypher = `
+          MATCH((message:Message)-[:INSIDE]->(:Room { id: $roomId })) RETURN COUNT(message) AS count
+          `
+          const countMessageTxResponse = await transaction.run(countMessageCypher, {
+            roomId,
+          })
+          return await countMessageTxResponse.records[0].get('count')
         })
```

Comment:
All this crap could be removed if we could get an `indexId` with numeric unique incrementing value on the message type. It is unclear if there can be holes in the index id or not.